openpocket/alpha_spheres/alpha_spheres.py

```diff
@@ -126,27 +125,44 @@ def remove_alpha_spheres(self, indices):
         --------
 
         """
-
-        pass
-
-    def remove_small_alpha_spheres(self, minimum_radius):
-
-        """
-
-        """
-
-        indices_to_remove = np.where(self.radii < minimum_radius)
-        self.remove(indices_to_remove)
+        if indices is not None:
+        #    if isinstance(indices, (list,np.ndarray,dtype:ints)): 
+            if isinstance(indices, (list,np.ndarray)):
+                pass
+            else:
+                raise ValueError("The argument points needs to be a List, tuple or numpy.ndarray object with the integer numbers corresponding to the alpha-sphere")
+
+            self.indices = indices  # a way to store the input indices
+            alpha_sphere_index=[]
+            for ia,alpsp in enumerate(self.points_of_alpha_sphere):
+                alpha_sphere_index.append(ia) 
+            for ix in self.indices:
+#                if ix in alpha_sphere_index:
+                alpha_sphere_index.remove(ix)
+               # remaining=[]
+                self.remove_alpha_spheres=[]
+                for ai in alpha_sphere_index:
+                    self.remove_alpha_spheres.append(self.points_of_alpha_sphere[ai])
+               #     self.remove_alpha_spheres = self.points_of_alpha_sphere[ai]
+               # else:
+               #     raise IndexError("has been using a non existent index")
 
```

Comment:
You didn't update the content of `self.centers`, `self.points_of_alpha_sphere`, `self.radii` and `self.n_alpha_spheres`.

Reply:
Another mistake. Actually, I don't know how to call a variable defined in another method.